### ISSUE TITLE
Adding documentation for using this plugin 

### ISSUE THREAD

**Issue by almothafar** [playframework/play-ebean#211]
Previously, there is documentation in the Play Framework website https://www.playframework.com/documentation/2.6.x/JavaEbean  
but when I use https://www.playframework.com/documentation/latest/JavaEbean it will redirect me to here, there is no reference and no documentation to how to use it now.

**Comment by felipebonezi** [comment]
@almothafar did you that exists this documentation markdown file?
https://github.com/playframework/play-ebean/blob/master/docs/manual/working/javaGuide/main/sql/JavaEbean.md

Anyway, I agree that we must make easy to know about how to use this plugin.
@SethTisue @octonato how do you feel about moving this file to `README.md` in root folder?

**Comment by octonato** [comment]
README is perfect.

**Comment by mreiche** [comment]
No, the linked `README` is far from perfect. It's outdated and just wrong.

```
Class 'MyServerConfigStartup' must either be declared abstract or implement abstract method 'onStart(DatabaseConfig)' in 'ServerConfigStartup'
```
```
import play.db.ebean.Transactional;
Cannot resolve symbol play.db.ebean.Transactional
```
There is not explanation how to configure Ebeans.

**Comment by felipebonezi** [comment]
I agree.

We must improve documentation to increase adopters and even maintainers for this project.

**Comment by plouf142** [comment]
Also, the readme file is made for compilation bot. The code snippets are'nt included, we have to do goings and comings with file linked. Read the markdown file in the github repository is only a workaround, not a solution, sorry!
